Source/Fuse.Controls.Panels/Layouts/WrapLayout.uno

```diff
@@ -82,21 +82,21 @@ namespace Fuse.Layouts
 		public string ID { get; set; }
 
 		internal override float2 GetContentSize(
-			IList<Node> elements,
+			Visual elements,
 			LayoutParams lp)
 		{
 			return Arrange(elements, lp, false);
 		}
 		
 		internal override void ArrangePaddingBox(
-			IList<Node> elements,
+			Visual elements,
 			float4 padding,
 			LayoutParams lp)
 		{
 			Arrange(elements, lp, true, padding);
 		}
 		
-		float2 Arrange(IList<Node> elements, LayoutParams lp,	
+		float2 Arrange(Visual elements, LayoutParams lp,	
```

Comment:
The name of the argument needs to change on these, `elements` is  misleading, especially when you see things like `elements.Children.Count` in the code.